pkg/mount/mount.go

```diff
@@ -4,6 +4,8 @@ import (
 	"sort"
 	"strings"
 
+	"syscall"
```

Comment:
In view of 069fdc8a083cb and similar commits, shouldn't we use `sys/unix` instead of `syscall` here and below?

Reply:
The main reason I chose syscall is because this is called from both Windows and Linux, both of which have EINVAL defined in syscall, but `unix.` is not compiled on Windows (and vice-versa).
We ultimately have the go1 guarantee on this and being it's not functionality which could be bugger but an error value I think we are fine here.

Reply:
Agreed